recipes/bigscape/meta.yaml

```diff
@@ -10,23 +10,20 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: "{{ PYTHON }} -m pip install --no-deps --ignore-installed . -vv"
 
 requirements:
   host:
-    - python ==3.6
+    - python ==3.7
   run:
-    - python ==3.6
+    - python ==3.7
```

Comment:
could you maybe remove these pins completely?

Reply:
I am not sure. The best is to ask the tool developer/maintainer: what do you think @jorgecnavarrom ? Could we use the last python release (currently 3.10) ?

Reply:
After discussion with @jorgecnavarrom, the answer is "no" (cannot remove the pins for the moment) because the other required packages have not been updated yet to be used with python 3.8+.